engine/src/test/java/org/hibernate/search/test/dsl/NumericTypeWithNullEncodingTest.java

```diff
@@ -36,7 +35,8 @@
 public class NumericTypeWithNullEncodingTest {
 
 	@Rule
-	public SearchFactoryHolder sfHolder = new SearchFactoryHolder( SomeEntity.class );
+	public SearchFactoryHolder sfHolder = new SearchFactoryHolder( SomeEntity.class )
+			.withProperty( org.hibernate.search.cfg.Environment.DEFAULT_NULL_TOKEN, "-7" );
```

Comment:
Should there be a property to give a default per field type? Or e.g. `Environment.DEFAULT_NUMERIC_NULL_TOKEN`. Applying a shared default for string and numeric fields appears problematic, as it likely forces people into overriding it in many cases.